### PR TITLE
Tempus-467, 468, 655

### PR DESCRIPTION
**Pull request by ganeshx**
Thank you for submitting a contribution to Tempus.

In order to streamline the review of the contribution please
ensure the following steps have been taken:

### For all changes:
- [x] Is there a Waffle ticket associated with this PR? Is it referenced 
     in the commit message?

- [x] Does your PR title start with Tempus-XXXX where XXXX is the waffle number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [x] Has your PR been rebased against the latest commit within the target branch (typically dev)?

- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [x] Have you ensured that the full suite of tests is executed via mvn clean install at the root Tempus folder?
- [ ] Have you written or updated unit tests to verify your changes?
- [x] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [ ] Have you added documentation for any UI or API related changes to the /docs folder

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by ganeshx** [comment]
WellLogViewer 21 and 23 are not relevant as they are rewritten in Tempus as part of #468. 
WellLogViewer  22, 24, 25, 26, 29, 31 are part of Tempus repository.

File: ui/src/app/widget/lib/well-log-viewer/well-log-viewer.js [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+          //  var settings = ctx.settings;
+            this.init();
+     }
+     init() {
+		//loadLogViewer(this.ctx, "init");
+        //alert(test);
+        //alert("test.test");
+     }
+
+     test(){
```
Comment (Shobhit2884) [inline ui/src/app/widget/lib/well-log-viewer/well-log-viewer.js]: Can we rename the name of the function?

File: ui/src/app/widget/lib/well-log-viewer/well-log-viewer.js [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+     	var datasources = this.ctx.datasources;
+        console.log(settings);
```
Comment (Shobhit2884) [inline ui/src/app/widget/lib/well-log-viewer/well-log-viewer.js]: Remove the console.log

File: ui/src/app/components/react/json-form-schema-form.jsx [hunk 3]
```diff
@@ -82,9 +84,16 @@ class TempusSchemaForm extends React.Component {
             console.log('Invalid field: \"' + form.key[0] + '\"!');
             return null;
         }
-        if(form.condition && eval(form.condition) === false) {
+ /*       if(form.condition && eval(form.condition) === false) {
```
Comment (Shobhit2884) [inline ui/src/app/components/react/json-form-schema-form.jsx]: Remove commented code

File: ui/src/app/widget/lib/well-log-viewer/dataGenerator.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+	let noise = new SimplexNoise();
+	let seed = 10 + 100 * Math.random();
+	dataGen.data = [seed];
+	let deltas = [seed];
+
+	dataGen.latestData = [seed];
+	let latestDeltas = [seed];
+
+	dataGen.tick = function() {
+	// d.getTime();
```
Comment (Shobhit2884) [inline ui/src/app/widget/lib/well-log-viewer/dataGenerator.js]: remove commented code

File: ui/src/app/widget/lib/well-log-viewer/headerLegend.js [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+ */
+import * as d3 from 'well-log-viewer/node_modules/d3/build/d3';
+import './logViewer.css';
+
+/*@ngInject*/
+
+var headerLegend = function(lineConfig, headerCount) {
+  'use strict';
+  var o;
+     // local;
```
Comment (Shobhit2884) [inline ui/src/app/widget/lib/well-log-viewer/headerLegend.js]: remove commented code

File: ui/src/app/widget/lib/well-log-viewer/headerLegend.js [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+    key: null,
+    min: lineConfig.headerMin,
+    max: lineConfig.headerMax,
+    lineWeight: lineConfig.lineWeight,
+    color: lineConfig.color,
+    label: lineConfig.headerName,
+    headerCount: headerCount
+  };  
+
+  // local = {
```
Comment (Shobhit2884) [inline ui/src/app/widget/lib/well-log-viewer/headerLegend.js]: remove commented code

File: ui/src/app/widget/lib/well-log-viewer/headerLegend.js [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+  // //  label: d3.local(),
+  // //  dimensions: d3.local()
+  // };
+ 
+  function header(group) {
+    // group-scope
```
Comment (Shobhit2884) [inline ui/src/app/widget/lib/well-log-viewer/headerLegend.js]: remove commented code

**Pull request merged**